assets/stylesheets/components/_app-container.scss

```diff
@@ -0,0 +1,3 @@
+.app-container {
```

Comment:
These styles should go in the layout as they affect how the page rendered.

Reply:
@olivierobert ✅  moved on [ababbb6](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/ababbb635ed9dacd53a66508770b72b570b7dec5)

Reply:
That's not what I meant 🙃 There should be a layout file e.g. `_application.scss`, `_authentication.scss`. Then in the template file, on teh root HTML element add a class name:

```
<htm class="layout-application">
```

And use in the screen stylesheet:

```
.layout-application {
  .app-header {}
  .app-container {}
  ...
}
```

Recheck how this is done on any past project if you are unsure